src/Engine/Utils.java

```diff
@@ -92,140 +92,136 @@ public static void highlightLocations(GameMap map, ArrayList<XYCoord> locations)
     }
   }
 
+  /** Alias for {@link #findPossibleDestinations(XYCoord, Unit, GameMap, boolean) findPossibleDestinations()} **/
```

Comment:
This link doesn't go to a real function.

Reply:
Where did these links come from, anyhow. Does Eclipse have a thing for making them?

Reply:
I don't think Eclipse auto-generates them (or how it would know where to point on its own) - it's just javadoc format to reference another part of the code. That way if you generate html docs or hover the symbol you get a nice hyperlink to follow.